test/runner/lib/executor.py

```diff
@@ -945,6 +956,15 @@ def command_integration_filtered(args, targets, all_targets, inventory_path, pre
 
     finally:
         if not args.explain:
+            if args.coverage:
+                coverage_temp_path = os.path.join(common_temp_path, COVERAGE_OUTPUT_PATH)
+                coverage_save_path = 'test/results/coverage'
+
+                for filename in os.listdir(coverage_temp_path):
+                    shutil.copy(os.path.join(coverage_temp_path, filename), os.path.join(coverage_save_path, filename))
```

Comment:
How large are these files? Should we just implement the remote get in this PR for when controller and remote is split?

Reply:
These coverage files are separate from what will eventually be the remote coverage files. We're using a common temp directory to accommodate tests which drop privileges and would thus be unable to write coverage files directly into our source tree.